src/client.ts

```diff
@@ -1,16 +1,17 @@
 import type { GlobalRootLinks, OctopusError, RootResource, SpaceRootLinks, SpaceRootResource } from "@octopusdeploy/message-contracts";
 import ApiClient from "./apiClient";
-import type { RouteArgs } from "./resolver";
-import type { Callback } from "./subscriptionRecord";
 import type { ClientConfiguration } from "./clientConfiguration";
+import { processConfiguration } from "./clientConfiguration";
```

Comment:
These should be able to be merged together into a single line and the `type` removed